packages/cli/src/commands/releasegold/revoke-votes.ts

```diff
@@ -0,0 +1,76 @@
+import { flags } from '@oclif/command'
+import BigNumber from 'bignumber.js'
+import { newCheckBuilder } from '../../utils/checks'
+import { displaySendTx } from '../../utils/cli'
+import { Flags } from '../../utils/command'
+import { ReleaseGoldCommand } from './release-gold'
+
+export default class RevokeVotes extends ReleaseGoldCommand {
+  static description = "Revokes votes for the given contract's account"
+
+  static flags = {
+    ...ReleaseGoldCommand.flags,
+    type: flags.string({
+      char: 'a',
+      options: ['active', 'pending'],
+      description: 'Type of votes to revoke [active, pending]',
+      required: true,
+    }),
+    group: Flags.address({
+      required: true,
+      description: 'Address of the group to revoke votes from',
+    }),
+    votes: flags.string({ required: true, description: 'The number of votes to revoke' }),
+  }
+
+  static examples = [
+    'revoke-votes --contract 0x47e172F6CfB6c7D01C1574fa3E2Be7CC73269D95 --type pending --group 0x5409ED021D9299bf6814279A6A1411A7e866A631 --votes 100',
```

Comment:
Is this what we do in the normal revoke? I think it's more user friendly for them to just provide a value, and then automatically revoke pending votes first, then active votes.

Reply:
You're right, I must have been asleep here, updated it